packages/components/src/organisms/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -88,28 +106,28 @@ function ImageGallerySelector({ images, onSelect, currentImageIdx }: Props) {
                 idx === currentImageIdx ? 'selected' : 'true'
               }
             >
-              <Image
-                onLoadingComplete={(img) => {
-                  if (ref) ref(img)
-                }}
-                src={image.url}
-                alt={image.alternateName}
+              <ImageComponent
+                url={image.url ?? ''}
                 loading={idx === 0 ? 'eager' : 'lazy'}
-                sizes="(max-width: 72px) 25vw, 30vw"
-                width={72}
-                height={72}
+                alternateName={image.alternateName ?? ''}
+                onLoad={(image) => ref && ref(image.currentTarget)}
               />
-            </UIButton>
+            </Button>
           )
         })}
       </div>
       {elementHasScroll && !lastImageInView && (
-        <UIIconButton
-          data-fs-image-gallery-selector-control-button
-          aria-label="forward slide image selector"
-          icon={<Icon name="ArrowLeft" width={24} height={24} />}
-          onClick={() => moveScroll(elementsRef.current, +SCROLL_MARGIN_VALUE)}
-        />
+        <div data-fs-image-gallery-selector-control>
+          <IconButton
+            data-fs-image-gallery-selector-control-button
+            aria-label="forward slide image selector"
+            icon={<ArrowLeft />}
+            size="small"
```

Comment:
Can this size be a prop too?

Reply:
Oh I think this one is fixed. It should small to work correctly.